docs/troubleshooting.md

```diff
@@ -85,6 +85,45 @@ $ export PATH=$PATH:/path/to/mpi/bin
 $ pip install --no-cache-dir horovod
 ```
 
+2. Are MPI libraries added to `$LD_LIBRARY_PATH` or `ld.so.conf`?
+
+If you see the error message below, it means `mpicxx` was not able to load some of the MPI libraries. If you recently
+installed MPI, make sure that path to MPI libraries are present the `$LD_LIBRARY_PATH` environment variable, or in the
```

Comment:
that path to --> that the path to
libraries are present --> libraries is present (unless it's multiple paths, in which case, it should say "that paths to MPI libraries are present")